__tests__/app.test.js

```diff
@@ -102,7 +103,34 @@ describe("GET /api", () => {
         .get("/api/articles/invalid")
         .expect(400)
         .then(({ body: { msg } }) => {
-          expect(msg).toBe("Bad request, expected number");
+          expect(msg).toBe("Bad request");
+        });
+    });
+  });
+
+  describe("/api/articles", () => {
+    test("200: Should return an array of article objects, sorted by created_at in descending order", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body: { articles } }) => {
+          expect(articles.length).toBe(13);
+          articles.forEach((article) => {
+            expect(article).toMatchObject({
+              author: expect.any(String),
+              title: expect.any(String),
+              article_id: expect.any(Number),
+              topic: expect.any(String),
+              created_at: expect.any(String),
+              votes: expect.any(Number),
+              article_img_url: expect.any(String),
+              comment_count: expect.any(String),
```

Comment:
Ideally a count should be a number. I'd also like to see an assertion that the body key isn't here.